fixed a crash (function call may happen on deleted object)

gondre opened the pull request.

sudara commented:
Thanks for the contribution! 

Could you provide some details about how to reproduce the crash?

gondre commented:
I just added that include to my PluginEditor.h:
`#include "melatonin_inspector/melatonin_inspector.h"` 

and the inspector to the members:
`melatonin::Inspector inspector { *this };`

The bug happened the first time I opened the plugin, using the vst3 in Reaper on macOS.
I fixed it immediately, so I'm not sure if it's reproducible all the time or not.
IIRC Reaper may create the editor twice in a row when instantiating the plugin, so I guess the callAfterDelay probably occurred right after the first created editor has been destroyed.

sudara commented:
Ok, thanks, that's very helpful!
